# -*- coding: utf-8 -*-
"""Zad.2

Automatically generated by Colaboratory.

Original file is located at
    https://colab.research.google.com/drive/1Xc_0fMvExW9RG5uC34S7XPH3ULvHBXWg
"""

def multiply_numbers(x, y):
    if not isinstance(x, int) or not isinstance(y, int):
        raise TypeError("Both arguments must be of type int")

    result = x * y
    return result

try:
    num1 = int(input("Podaj pierwszą liczbę całkowitą: "))
    num2 = int(input("Podaj drugą liczbę całkowitą: "))

    multiplication_result = multiply_numbers(num1, num2)
    print(f"Wynik mnożenia: {multiplication_result}")
except ValueError:
    print("Podano nieprawidłową liczbę całkowitą.")
except TypeError as e:
    print(f"Błąd: {e}")